JSTask5.js

```diff
@@ -0,0 +1,169 @@
+// Task 1
+// Remove duplicates of the array and return an array of only unique elements.
+
+const array = [1, 2, 3, 5, 1, 5, 9, 1, 2, 8];
+let dup = [...new Set(array)];
+console.log(dup);
+
+
+// Task 2
+/* Find the intersection of two arrays. (An intersection would be the common elements that exist within both arrays).
+The elements should be unique! */
+
+const firstArray = [2, 2, 4, 1];
+const secondArray = [1, 2, 0, 2];
+let intersection = firstArray.filter(value=>secondArray.includes(value));
+console.log(intersection);
+
+
+// Task 3
+// Remove empty values from the array.
+
+const array1 = [0, 1, null, 2, "", 3, undefined, 3,,,,,, 4,, 4,, 5,, 6,,,,];
+let filtered = array1.filter((el)=>{
+    return el !== null && typeof el !== 'undefined' && el !== '';
+});
+console.log(filtered);
+
+
+// Task 4
+//Write a JavaScript program to display whether the player is retired (i.e. display player name, nationality, and playing status) of the following players.
+
+const cricket = [
+    {
+        name: 'Sachin Tendulkar',
+        nationality: 'India',
+        retired: true
+    },
+    {
+        name: 'Shane Warne',
+        nationality: 'Australia',
+        retired: true
+    },
+    {
+        name: 'Virat Kohli',
+        nationality: 'India',
+        retired: false
+    }
+];
+
+// let cricket1 = [];
+// cricket1 = cricket.filter(c=>c.retired===true);
+// console.log(cricket1);
+
+console.log(cricket.filter(c=>c.retired===true));
+
+
+// Task 5
+/* Write a JavaScript function to get the first element of an array. 
+Passing a parameter 'n' will return the first 'n' elements of the array:
+Test Data :
+console.log(first([7, 9, 0, -2]));
+console.log(first([],3));
+console.log(first([7, 9, 0, -2],3));
+console.log(first([7, 9, 0, -2],6));
+console.log(first([7, 9, 0, -2],-3));
+Expected Output :
+7
+[]
+[7, 9, 0]
+[7, 9, 0, -2]
+[] */
+
+let ele = function(array,n){
+    if(array==null)
+    {
+        return 0;
+    }
+    if(n==null)
+    {
+        return array[0];
+    }
+    if(n<0)
+    {
+        return [];
+    }
+    return array.slice(0,n);
+}
+console.log(ele([7,9,0,-2]));
+console.log(ele([],3));
+console.log(ele([7, 9, 0, -2],3));
+console.log(ele([7, 9, 0, -2],6));
+console.log(ele([7, 9, 0, -2],-3));
+
+
+// Task 6
+// To find whether the element is present in the array or not.
+
+// Using indexOf() method.
+let pele = ["Aryann",20,"!","Lakshya","~","Karma",10,"Affan","@"];
+console.log(pele.indexOf("aryann"));
+console.log(pele.indexOf("Aryann"));
+
+// Using includes() method.
+console.log(pele.includes("!"));
+console.log(pele.includes("affan"));
+
+
+// Task 7
+// Remove the last element of an array (without using slice).
+const a1 = [1, 2, 3];
+console.log(a1.pop());
+
+
+// Task 8
+// Check If at Least one Array Element is greater than 5.
```

Comment:
Question expects a boolean output. Current output is an array of numbers greater than 5. Please code according to the question